BParkingNano/plugins/ElectronMerger.cc

```diff
@@ -49,7 +49,8 @@ class ElectronMerger : public edm::global::EDProducer<> {
     etaMax_{cfg.getParameter<double>("etaMax")},
     bdtMin_{cfg.getParameter<double>("bdtMin")},
     use_gsf_mode_for_p4_{cfg.getParameter<bool>("useGsfModeForP4")},
-    sortOutputCollections_{cfg.getParameter<bool>("sortOutputCollections")} 
+    sortOutputCollections_{cfg.getParameter<bool>("sortOutputCollections")},
+    saveLowPtE_{cfg.getParameter<bool>("saveLowPtE")}
```

Comment:
here you mean "useLowPtE" right?
it's not only saving them as Electrons in the ntuples, but also not using them at all for creating B candidates, right?
what would be the use case? 
do you know if the current code complains when the lowPt collection is not present in the input data?

Reply:
Yes exactly. Not saving them, not using them as well. The idea is if we want to check something fast on electrons or if there is an improvement only in PF (eg an update on the PF ID). Of course similar flag can be added for only low pT or low pT and PF. I started with PF since is simpler and cleaner. Running without low pT it will crash. It needs an input and since ID also uses stuff it is a bit complicated to remove it by comments. Also I think adding very high ID cut is a bit messy down the line